src/redux/store/reducer.js

```diff
@@ -2,7 +2,7 @@ import { combineReducers } from 'redux';
 
 
 const initialState = {
-  user: 'StormWingDelta',
+  user: 'alexmobi',
   albumList: [],
   imageList: []
 }
```

Comment:
could you explain   
> const newState = {...state};

* reducer should be implemented with switch cases and each case should return proper state
*

Reply:
Done.